engine/src/components/formcomponent.ts

```diff
@@ -2,7 +2,12 @@ import Component from "./component";
 import { optionalText } from "./constants";
 import joi from "joi";
 
+import { ViewModel } from "./types";
+
 export default class FormComponent extends Component {
+  isFormComponent: boolean;
```

Comment:
is it a safe assumption to just have this as `isFormComponent = true` (since the class _is_ FormComponent)